examples/knative/powershell/kn-ps-webhook/test/send-cloudevent-test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+WEBHOOK_USERNAME='FILL_IN_WEBHOOK_USERNAME'
+WEBHOOK_PASSWORD='FILL_IN_WEBHOOK_PASSWORD'
+
+### DO NOT EDIT BEYOND HERE ###
+
+WEBHOOK_BASE64=$(echo -n "${WEBHOOK_USERNAME}:${WEBHOOK_PASSWORD}" | openssl base64)
```

Comment:
Is the `base64` binary not installed in our base image?

Reply:
This is for local development. I found some oddities simply relying on `base64` on MacOS and most systems will have openssl and found that to be more reliable